docs/interfaces/pyapi/pyapi.rst

```diff
@@ -0,0 +1,425 @@
+
+    Hamiltonian = DFTB {
+
+      Scc = Yes
+      MaxSccIterations = 100
+      SccTolerance = 1.00e-05
+
+      SlaterKosterFiles = Type2FileNames {
+	Prefix = "./slakos/mio-ext/"
+	Separator = "-"
+	Suffix = ".skf"
+      }
+
+      MaxAngularMomentum {
+	Ti = "d"
+	O  = "p"
+      }
+
+      KPointsAndWeights = SupercellFolding {
+	4   0   0
+	0   4   0
+	0   0   4
+	0.5 0.5 0.5
+      }
+
+    }
+
+To be able to also extract the forces via the interface, it is advisable to
+specify the corresponding entry in the ``Analysis{}`` block. Otherwise DFTB+
+will issue an error message, asking the user to do so. To ensure backwards
+compatibility of the input, the parser version is also specified::
+
+    Analysis {
+      CalculateForces = Yes
+    }
+
+    ParserOptions {
+      ParserVersion = 8
+    }
+
+
+.. |TiO2| replace:: TiO\ :sub:`2`\
+
+.. _sec-interfaces-pyapi-mainscript:
+
+Main script
+-----------
+
+The script shown here serves to illustrate the use of the interface, based on
+the calculation of |TiO2|.
+
+In order to be able to use the interface, the package `dftbplus` must be
+imported as the first step. The path `LIB_PATH` to the shared library is
+defined, as well as conversion factors to convert the atom coordinates
+from Ångström into atomic units (Bohr), required by the interface.
+
+.. code-block:: python
+
+    import numpy as np
+    import dftbplus
+
+
+    LIB_PATH = '/home/user/libdftbplus'
+
+    # DFTB+ conversion factors
+    # (according to prog/dftb+/lib_common/constants.F90)
+    BOHR__AA = 0.529177249
+    AA__BOHR = 1 / BOHR__AA
+
+At the beginning of the ``main()`` function, the atom coordinates and lattice
+vectors are defined. In this case, a conversion to atomic units is
+necessary, since a `.gen` block is used whose values usually have the unit
+Ångström.
+
+.. _sec-interfaces-pyapi-codeblock1:
+
+.. code-block:: python
+
+    def main():
+	'''Main driver routine.'''
+
+	# coordinates of TiO2, in Ångström
+	coords = np.array([
+	    [-0.016726922839251,  0.016725329441158, -0.000003204152532],
+	    [-0.016726505918979,  1.920201169305565, -7.297102897292027],
+	    [ 0.017412997824265, -0.024318617967798,  2.005339137853385],
+	    [ 1.920770753428742, -0.024319922392223, -4.437737763954652],
+	    [ 0.024319174400169, -0.017404302527510, -2.005347277168561],
+	    [ 0.024317270342179,  1.886164739806594, -5.291732430733527]])
+
+	# lattice vectors of TiO2, in Ångström
+	latvecs = np.array([
+	    [-1.903471721000000,  1.903471721000000,  4.864738245000000],
+	    [ 1.903471721000000, -1.903471721000000,  4.864738245000000],
+	    [ 1.903471721000000,  1.903471721000000, -4.864738245000000]])
+
+	# conversion to atomic units
+	coords *= AA__BOHR
+	latvecs *= AA__BOHR
+
+An object of the DftbPlus class is instantiated, whereby the path to the
+shared library `libpath`, the path to the HSD input file `hsdpath` and the
+name of the log file `logfile` can be specified. These are optional keyword
```

Comment:
reStructuredText is not markdown, in rst two backticks are required for literals:
```suggestion
shared library ``libpath``, the path to the HSD input file ``hsdpath`` and the
name of the log file ``logfile`` can be specified. These are optional keyword
```

Reply:
I just wanted to italicize paths and occasionally also variables (but not as monospaced text). It looks like the possibilities to achieve this have changed at some point. At least some of the recipe entries use single backticks as the method of choice, as far as I've seen it. I did not want to break the convention here.